R/plot-population-time-profile.R

```diff
@@ -0,0 +1,132 @@
+#' Concentration time profile plot for population
+#'
+#' @inheritParams plotIndividualTimeProfile
+#' @param quantiles A numerical vector with quantile values (Default: `c(0.05,
+#'   0.50, 0.95)`). In the profile plot, the middle value will be used to draw a
+#'   line, while the lower and upper values will be used to create a ribbon.
+#'
+#' @import tlf
+#'
+#' @family plotting
+#'
+#' @examples
+#'
+#' # TODO: add example
```

Comment:
PLease create an issue for documentation of this function.

Reply:
Done.

#927